std/mmfile.d

```diff
@@ -134,18 +134,20 @@ class MmFile
             char c = 0;
             core.sys.posix.unistd.write(fd, &c, 1);
         }
-        else if (prot & PROT_READ && size == 0)
+        else if (prot & PROT_READ && size == 0) {
+            if(statbuf.st_size == 0) return;
             size = statbuf.st_size;
+        }
         this.size = size;
 
         // Map the file into memory!
         size_t initial_map = (window && 2*window<size)
             ? 2*window : cast(size_t) size;
         auto p = mmap(address, initial_map, prot, flags, fd, 0);
         if (p == MAP_FAILED)
-        {
-            errnoEnforce(false, "Could not map file into memory");
-        }
+            {
+                errnoEnforce(false, "Could not map file into memory");
+            }
```

Comment:
Either use Allman or egyptian braces, but not GNU. Consistency makes the code much more readable. In this case especially since there are no actual change here.